stubs/Send2Trash/send2trash/__main__.pyi

```diff
@@ -0,0 +1,5 @@
+from collections.abc import Iterable
+
+from send2trash import send2trash as send2trash
```

Comment:
We probably don't need the re-export here?

```suggestion
```

Reply:
stubgen "features" :)